aladdin-container.sh

```diff
@@ -161,6 +168,61 @@ function _add_authentication_user_to_kubeconfig() {
 EOT
 }
 
+function _handle_aws_config() {
+    # This function does quite a few things with aws credentials/config files:
+    # 1) Move the mounted aws creds to ~/.aws
+    # 2) Checks if we have the BASTION_ACCOUNT_ENABLED configuration from our aladdin-config
+    # 3) If we do:
+    #        - export appropriate BASTION_ variables for other aladdin commands
+    #        - sanity test our aws configuration for the bastion account if INIT and not IS_LOCAL
+    #        - appropriately update our credentials/config files with the desired possible roles
+    #            - here we are calling the add-aws-assume-role-config aladdin command
+    # 4) If we do not:
+    #        - sanity test our aws configuration for the cluster's current aws account if INIT and not IS_LOCAL
+    # 5) Export AWS_PROFILE=$AWS_DEFAULT_PROFILE because kops uses the former
+
+    # Move aws credentials away from mount so we don't edit the host's aws files
+    cp -r /root/tmp/.aws /root/.aws
+    # See if bastion account is enabled
+    export BASTION_ACCOUNT_ENABLED="$(_extract_cluster_config_value bastion_account_enabled)"
+    if "$BASTION_ACCOUNT_ENABLED"; then
+        # Export BASTION_ACCOUNT_PROFILE, needed by add-aws-assume-role-config command
+        export BASTION_ACCOUNT_PROFILE="$(_extract_cluster_config_value bastion_account_profile)"
+        # Need to unset AWS_DEFAULT_PROFILE because aws requires that entry to exist even if we
```

Comment:
This was an annoying issue. see the comment by billwanjohi here for more info https://stackoverflow.com/questions/34134879/aws-the-config-profile-myname-could-not-be-found